Project.toml

```diff
@@ -13,6 +13,7 @@ Metalhead = "dbeba491-748d-5e0e-a39e-b530a07fa0cc"
 Optimisers = "3bd65402-5787-11e9-1adc-39752487f4e2"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+ScientificTypes = "321657f4-b219-11e9-178b-2701a2544e81"
```

Comment:
Have you decided it's not possible to demote this requirement to ScientificTypesBase.jl? Or is that on the to-do list?

Reply:
Yes, on the todo (it just got added after I was about finished); will do it as a response to this item in the review.

Reply:
Done.